express-hello-world/index.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const app = express();
+// eslint-disable-next-line no-unused-vars
```

Comment:
Maybe you should have paid attention to the lint warning! ;-)
You should change line 19 so that it uses this constant (or get rid of the constant).

Reply:
so true..... :(